ts-cloudfs/index.ts

```diff
@@ -23,6 +26,7 @@ app.get('/', async (req, res) => {
 
 /* @klotho::expose {
  *  target = "public"
+ *  id = "app"
```

Comment:
I'm not sure how much we care about this level of detail, but these new directives have different indentation than the `name` ones that you changed to `id`.